data/tags.json

```diff
@@ -336,10 +338,22 @@
     "content": ["https://tryitands.ee/"]
   },
   { "name": "vim4newcomers",
-    "aliases": [],
+    "aliases": ["vim"],
     "content": [
-      "The best way to get started with vim is using `vimtutor`, which is pre-included with vim / neovim!",
-      "If you're looking for some pre-configured framework with sane defaults and plugins, [LunarVim](https://github.com/LunarVim/LunarVim) might be for you!"
+      "**The best way to learn vim is using tutor, which is pre-included with vim/neovim!**",
```

Comment:
nit: "The best way to learn vim is by using the tutor mode that's included with vim and neovim."?